src/main/kotlin/org/partiql/scribe/targets/spark/SparkAstToSql.kt

```diff
@@ -116,7 +117,12 @@ public open class SparkAstToSql(context: ScribeContext) : AstToSql(context) {
      * INT2 -> SMALLINT
      * INT4 -> INT
      * INT8 -> BIGINT
-     * DOUBLE_PRECISION -> DOUBLE
+     * DOUBLE PRECISION -> DOUBLE
+     * VARCHAR -> no params given default of 20 (https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/types/VarcharType.html#defaultSize())
+     * CHAR -> no params given default of 20 (https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/types/CharType.html#defaultSize())
```

Comment:
self-review: in SparkSQL, a parameter must be provided for VARCHAR and CHAR. Using the default of 20 for now.